src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -1244,7 +1244,7 @@
     <value>Invalid type for ParameterInfo member in Attribute class.</value>
   </data>
   <data name="Argument_InvalidPathChars" xml:space="preserve">
```

Comment:
(nit) do we want to change the name of this resource string so someone doesn't mistakenly use it when they are checking for characters other than `\0`?